common/src/main/resources/less/style.less

```diff
@@ -62,6 +62,15 @@ body {
   background: darkgrey;
 }
 
+// Taken from the definitíon of ui.button transition but removed background-color and background
+// When animating those two properties you see some flickering in FF
```

Comment:
Does this apply only to side buttons or to all buttons?

Reply:
Only side buttons, it looks fine in regular buttons